src/spanningtrees/boruvka.jl

```diff
@@ -0,0 +1,73 @@
+"""
+    boruvka_mst(g, distmx = weights(g); minimize = true)
+
+Return a tuple `(mst, weights)` where `mst` is a vector of edges representing the
+optimum (minimum, by default) spanning tree of a connected, undirected graph
+`g` with optional matrix `distmx` that provides distinct edge weights, and
+`weights` is the sum of all the edges in the solution by using
+[Boruvka's algorithm](https://en.wikipedia.org/wiki/Bor%C5%AFvka%27s_algorithm).
+The algorithm requires that all edges have different weights to correctly generate a minimun/maximum spanning tree
+### Optional Arguments
+- `minimize=true`: if set to `false`, calculate the maximum spanning tree.
+"""
+
+function boruvka_mst end 
+
+@traitfn function boruvka_mst(graph::AG::(!IsDirected), 
+        distmx::AbstractMatrix{T} = weights(graph); 
+        minimize = true) where {T<:Real,U,AG<:AbstractGraph{U}}
+
+    djset = IntDisjointSets(nv(graph))
+    # maximizing Z is the same as minimizing -Z
+    # mode will indicate the need for the -1 multiplication
+    mode = (-1)^(minimize ? 0 : 1)
+
+    mst = Vector{edgetype(graph)}()
+    sizehint!(mst, nv(graph) - 1)
+    weight = zero(T)
+
+    while true
+    
```

Comment:
No need for a blank line here.

Reply:
line 28? I think it doesn't look great this way

Reply:
Are you sure? I can change it if it's better, but it looks kinda strange.

Reply:
I've done the other changes already, just waiting the answer

Reply:
I've changed it so you can take a look, I can change it back.

Reply:
I think blank lines L29, L36, L57, L59 should go.